src/app/public/modules/flyout/flyout.service.ts

```diff
@@ -104,7 +113,7 @@ export class SkyFlyoutService implements OnDestroy {
       .fromEvent(windowObj, 'click')
       .takeUntil(this.idled)
       .subscribe((event: MouseEvent) => {
-        if (this.host && this.host.location && !this.host.location.nativeElement.contains(event.target)) {
+        if (this.adapter.shouldClickCloseFlyout(this.host, <Node> event.target)) {
```

Comment:
As much as possible, Angular components should be unaware of the DOM API. We should change the `event: MouseEvent` to `event: any`, and pass that object to the adapter directly.

```
.subsribe((event: any) => {
  if (this.adapter.hostContainsEventTarget(this.host, event)) {
    // ...
  }
```

Reply:
Good thought. Fixed